third_party/efr32_sdk/efr32_board.gni

```diff
@@ -57,6 +57,9 @@ if (efr32_board == "BRD4304A") {
 } else if (efr32_board == "BRD4180A") {
   efr32_family = "efr32mg21"
   efr32_mcu = "EFR32MG21A020F1024IM32"
+} else if (efr32_board == "BRD4308A") {
+  efr32_family = "efr32mg21"
+  efr32_mcu = "MGM210PA22JIA"
```

Comment:
I don't think this is the MCU name. MGM210PA22JIA is the module name but I didn't validate.